sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultStep.java

```diff
@@ -139,6 +139,9 @@ public String getName() {
     @Override
     public Status getStatus() {
         synchronized (statusLock) {
+            if (status == Status.PENDING && getStrategy().isInterrupted()) {
+                return Status.WAITING;
```

Comment:
interrupt (pause) and status are independent. Only step has status variable and setStatus method.  Step is the last one in PlanUtil.getStatus lookup chain.  This PR will make output (plan show) show that Step is waiting. Other than that it should have no effect.